src/app/app-design-tool/types.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+import * as types from '../app-interactive-map/types';
+
+export interface Product {
+  name: string;
+  id: string;
+}
+
+export interface Design {
+  species: types.Species;
+  product: Product;
+  bigg: boolean;
+  kegg: boolean;
+  rhea: boolean;
+  model: types.DeCaF.Model;
+  number_pathways: 10;
```

Comment:
this is not an initial value, but you hardcode it to 10. If that's not what you want, you can use a class with some default values in the constructor.